pallet-chainlink-feed/src/tests.rs

```diff
@@ -1210,7 +1210,11 @@ fn can_go_into_debt_and_repay() {
 		assert_eq!(Balances::free_balance(admin), new_funds - 10);
 		assert_eq!(ChainlinkFeed::debt(0).unwrap(), payment - 10);
 		// should be possible to overshoot in passing the amount correcting debt...
-		assert_ok!(ChainlinkFeed::reduce_debt(Origin::signed(42), 0, payment));
+		assert_ok!(ChainlinkFeed::reduce_debt(
```

Comment:
Maybe add an extra test @mattsse that shows/asserts that non-admins cannot reduce dept?

Reply:
Added a check against non admin